src/parser.ts

```diff
@@ -0,0 +1,37 @@
+import semver from "semver";
```

Comment:
do you think I should make some tests for these or is the integration test we have enough?

Reply:
We can trust the lib to be tested and working as expected! We could have a very simple test to make sure we use the lib properly but might not be the biggest priority for now!